### PR TITLE
Update set-up-irm-in-sp-admin-center.md

### PR DESCRIPTION
**Pull request by karthik55217**
We updated PDF protector in SharePoint IRM, and updating the doc accordingly.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @karthik55217, not related to this PR but kindly also commit the minor suggested change to correct the typo. Many thanks.

File: microsoft-365/compliance/set-up-irm-in-sp-admin-center.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ When people download files in an IRM-enabled list or library, the files are encr
   
 You cannot create or edit documents in an IRM-enabled library using Office in a browser. Instead, one person at a time can download and edit IRM-encrypted files. Use check-in and check-out to manage  *co-authoring*  , or authoring across multiple users.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/set-up-irm-in-sp-admin-center.md]:
```suggestion
You cannot create or edit documents in an IRM-enabled library using Office in a browser. Instead, one person at a time can download and edit IRM-encrypted files. Use check-in and check-out to manage  *co-authoring*, or authoring across multiple users. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by karthik55217** [comment]
@KCCross @JohanFreelancer9  - I committed the suggestions. Could you help merge this PR into main branch? (Looks like I do not have permissions to merge)

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder.

**Pull request merged**